.github/workflows/release.yaml

```diff
@@ -4,16 +4,15 @@ on:
   workflow_run:
     workflows: [Build]
     types: [completed]
-    branches-ignore:
-      - release
 
 jobs:
   release:
     name: Create release
     runs-on: ubuntu-latest
     if: |
       github.event.workflow_run.conclusion == 'success' &&
-      github.event.workflow_run.event != 'schedule'
+      github.event.workflow_run.event == 'push' &&
+      startsWith(github.event.workflow_run.head_branch, 'v')
```

Comment:
perhaps:
```
github.event.workflow_run.head_branch matches '^v\d+(\.\d+)*$'
```
?

Reply:
Supposedly regex not allowed (sources: https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#startswith https://github.com/orgs/community/discussions/26186) , but you drew my attention to the fact that it needs to be `refs/tags/v` instead of `v`